.travis.yml

```diff
@@ -1,17 +1,30 @@
 dist: xenial
 language: python
-python:
-  - "3.7"
 
-install:
-  - pip install --upgrade pip
-  - pip install -U -r python/requirements.txt dask[dataframe] distributed
+jobs:
+  include:
+    - language: python
+      python:
+        - "3.7"
+      install:
+        - pip install --upgrade pip
+        - pip install -U -r python/requirements.txt dask[dataframe] distributed
+      script:
+        - cd python && python setup.py develop test
+        - python setup.py clean
+        - rm -rf fbprophet/stan_model
+        - wget https://github.com/stan-dev/cmdstan/releases/download/v2.22.1/cmdstan-2.22.1.tar.gz -O /tmp/cmdstan.tar.gz > /dev/null
+        - tar -xvf /tmp/cmdstan.tar.gz -C /tmp > /dev/null
+        - make -C /tmp/cmdstan-2.22.1/ build > /dev/null
+        - CMDSTAN=/tmp/cmdstan-2.22.1 STAN_BACKEND=CMDSTANPY python setup.py develop test
 
-script:
-  - cd python && python setup.py develop test
-  - python setup.py clean
-  - rm -rf fbprophet/stan_model
-  - wget https://github.com/stan-dev/cmdstan/releases/download/v2.22.1/cmdstan-2.22.1.tar.gz -O /tmp/cmdstan.tar.gz > /dev/null
-  - tar -xvf /tmp/cmdstan.tar.gz -C /tmp > /dev/null
-  - make -C /tmp/cmdstan-2.22.1/ build > /dev/null
-  - CMDSTAN=/tmp/cmdstan-2.22.1 STAN_BACKEND=CMDSTANPY python setup.py develop test
+    - language: r
+      r:
+        - devel
+      install:
+        - R -e 'install.packages("remotes")'
```

Comment:
for some reason it wasn't picking up remotes or devtools automatically unless i installed it - i though the r devel version should already have that installed and available in namespace ?